src/main.rs

```diff
@@ -33,6 +33,26 @@
 
 //! Parsec Tool: a tool to communicate with Parsec from the command-line
 
+#[macro_use]
+mod output;
+
+mod cli;
+mod command;
+mod common;
+mod error;
+
+use crate::command::dispatch_command;
+use anyhow::{Context, Result};
+
+fn run() -> Result<bool> {
```

Comment:
Is the `bool` result actually used? If not, this function can maybe return `Result<()>`.

Reply:
No, not yet. The idea was going to be:

* Return `Err` on complete failure.
* Return `Ok(true)` on success with partial failure.
* Return `Ok(false)` on complete success.

But since we aren't using this right now, I agree -- better to just remove it.